src/examples/render.ts

```diff
@@ -0,0 +1,56 @@
+import { Renderer } from '../renderer/Renderer';
+
+import { mat4 } from 'gl-matrix';
+import { range } from 'lodash';
+
+const renderer = new Renderer(800, 600);
+document.body.appendChild(renderer.stage);
+
+const transform = mat4.create();
+mat4.translate(transform, transform, [0, 0, -8]);
```

Comment:
Could we declare all of the global `const`s at the top of the file, and do the other logic proceeding?

i.e.

```ts
const renderer = new Renderer(800, 600);
const transform = mat4.create();
const vertices: number[] = [];
const normals: number[] = [];
const indices: number[] = [];
const numLat = 20;
const numLong = 20;

document.body.appendChild(renderer.stage);
mat4.translate(transform, transform, [0, 0, -8]);

range(numLat).forEach((lat: number) => {
  // ...
```

So in terms of TypeScript modules, it would be:

1. imports
2. global `const`s
3. global logic
4. functions